### PR TITLE
Problem: unnecessary use of REST API instead of grpc-web (`get_account_balance` part)

### PR DESCRIPTION
**Pull request by stevenatcrypto**
Part of issue https://github.com/crypto-com/defi-wallet-core-rs/issues/511 (will fix `get_account_details` in another PR)

Replace REST API with `grpc-web` for `get_account_balance` functions.

# PR Checklist:

- [x] Have you read the [CONTRIBUTING.md](https://github.com/crypto-com/defi-wallet-core-rs/blob/main/CONTRIBUTING.md)?
- [x] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [x] Have you rebased your work on top of the latest main? 
- [x] Have you checked your code compiles? (`cargo build`)
- [x] Have you included tests for any non-trivial functionality?
- [x] Have you checked your code passes the unit tests? (`cargo test`)
- [x] Have you checked your code formatting is correct? (`cargo fmt -- --check --color=auto`)
- [x] Have you checked your basic code style is fine? (`cargo clippy`)
- [x] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`cargo audit`)
- [x] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [x] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-com/defi-wallet-core-rs/blob/main/CHANGELOG.md)?
- [x] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-com/defi-wallet-core-rs/blob/main/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-com/defi-wallet-core-rs/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: example/android_example/app/src/main/java/com/example/android_example/MainActivity.kt [hunk 1]
```diff
@@ -43,10 +43,10 @@ class MainActivity : AppCompatActivity() {
             text += h
         }
 
-        val rb = getAccountBalanceBlocking("https://mainnet.crypto.org:1317","cro1yjjlx5qsrj5rxn5xtd5rkm6dcqzlchxkrvsmg6","basecro",BalanceApiVersion.NEW)
+        val rb = getAccountBalanceBlocking("https://mainnet.crypto.org:1317","cro1yjjlx5qsrj5rxn5xtd5rkm6dcqzlchxkrvsmg6","basecro")
```
Comment (tomtau) [inline example/android_example/app/src/main/java/com/example/android_example/MainActivity.kt:46]: would this example work? that URL is for REST, not grpc?

**Review by tomtau — APPROVED** [review]
looks ok, but some binding examples (android, JS/Wasm...) may still have old URLs?

**Review by damoncro — APPROVED** [review]

File: example/android_example/app/src/main/java/com/example/android_example/MainActivity.kt (hunk 1 above)
Comment (tomtau) [reply example/android_example/app/src/main/java/com/example/android_example/MainActivity.kt]: how about the JS example?

File: example/js-example/index.js [hunk 2]
```diff
@@ -48,7 +48,7 @@ logWalletAddresses(wallet);
 // const account = await wasm.query_account_details("https://testnet-croeseid-4.crypto.org:1317", "tcro1y6493k3smakl2wf09u7ds4amztx8ks7leyrtmy");
 // console.log(account);
 
-// const balance = await wasm.query_account_balance("https://testnet-croeseid-4.crypto.org:1317", "tcro1y6493k3smakl2wf09u7ds4amztx8ks7leyrtmy", "basetcro", 0);
+// const balance = await wasm.query_account_balance("https://testnet-croeseid-4.crypto.org:9090", "tcro1y6493k3smakl2wf09u7ds4amztx8ks7leyrtmy", "basetcro");
```
Comment (tomtau) [inline example/js-example/index.js:51]: 9090 is grpc, not grpc-web?

**Pull request merged**